plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/CDKDockerUtility.java

```diff
@@ -54,7 +54,7 @@ public String getNextName(IServer server) {
 		return name;
 	}
 	
-	public IDockerConnection findDockerConnection(ADBInfo adb) {
+	public IDockerConnection findDockerConnection(ServiceManagerEnvironment adb) {
```

Comment:
Just a trifle remark. ADBInfo is renamed into ServiceManagerEnvironment but (here and in other places) variable name is 'adb' which will put before future generations a mystery to guess over: 'why adb'? :)